transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/auth/AuthHandlerImpl.kt

```diff
@@ -51,6 +51,21 @@ internal class AuthHandlerImpl(
         }
     }
 
+    override fun authorizeImplicit(idToken: String) {
+        dispatcher.launch {
+            when (val result = api.fetchAuthJwt(idToken)) {
+                is Result.Success -> {
+                    result.value.let {
+                        authJwt = it
+                        vault.authRefreshToken = NO_REFRESH_TOKEN
```

Comment:
Just wondering why we set the authRefreshToken to NO_REFRESH_TOKEN upon success.